app/views/projects/tree/show.html.haml

```diff
@@ -3,12 +3,22 @@
 = content_for :meta_tags do
   - if current_user
     = auto_discovery_link_tag(:atom, namespace_project_commits_url(@project.namespace, @project, @ref, format: :atom, private_token: current_user.private_token), title: "#{@project.name}:#{@ref} commits")
-
 = render 'projects/last_push'
 
 - if can? current_user, :download_code, @project
```

Comment:
I don't think we need this check here at all, since the TreeController already verifies this.